### PR TITLE
SJQ2T-321: BE - Permissions For Get Bucket Details

### PR DESCRIPTION
**Pull request by DarkoVeljanoskiHtec**

### CODE REVIEW

File: backend/src/main/java/com/htecgroup/skynest/service/impl/BucketServiceImpl.java [hunk 1]
```diff
@@ -71,6 +68,7 @@ public BucketResponse createBucket(BucketCreateRequest bucketCreateRequest) {
 
   @Override
   public BucketResponse getBucketDetails(UUID uuid) {
+    permissionService.currentUserHasPermissionForBucket(uuid, AccessType.VIEW);
```
Comment (DraganMitrasinovicHtec) [inline backend/src/main/java/com/htecgroup/skynest/service/impl/BucketServiceImpl.java:71]: Is this gohna work if he has a higher access type?

Comment (DarkoVeljanoskiHtec) [reply backend/src/main/java/com/htecgroup/skynest/service/impl/BucketServiceImpl.java:71]: I'm pretty sure yes, see the currentUserHasPermissionForBucket method.

Comment (DraganMitrasinovicHtec) [reply backend/src/main/java/com/htecgroup/skynest/service/impl/BucketServiceImpl.java:71]: My bad, didn't see the function accepts minimumAccessType, then I believe this is good

**Review by DraganMitrasinovicHtec — APPROVED** [review]
Lovely

File: backend/src/main/java/com/htecgroup/skynest/service/impl/BucketServiceImpl.java (hunk 1 above)
Comment (DarkoVeljanoskiHtec) [reply backend/src/main/java/com/htecgroup/skynest/service/impl/BucketServiceImpl.java:71]: It's written by @ivanfhtec, of course it's good.

**Pull request merged**